### PR TITLE
Add FAQ entries. Closes #1166, #1582. Contributes to #1587

### LINKED ISSUE #1166: Backup to OneDrive or Google Drive
Is it possible to back up to OneDrive or Google Drive? I have tried Rclone and Insync. Backintime cannot recognise the drives mapped by them.

### PR DESCRIPTION
**Pull request by aryoda**
- Corrects some other typos in code comments
- Add global locking details to dev doc of control files usage

Don't forget the changelog entry! ;)

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -1,51 +1,100 @@
...
+
+<!-- TOC start (generated with https://github.com/derlin/bitdowntoc) -->
+
 - [General](#general)
-  * [Does Back In Time create incremental backups?](#does-back-in-time-create-incremental-backups)
-  * [How does snapshots with hard-links work?](#how-does-snapshots-with-hard-links-work)
-  * [How can I check if my snapshots are using hard-links?](#how-can-i-check-if-my-snapshots-are-using-hard-links)
-  * [How to use checksum to find corrupt files periodically?](#how-to-use-checksum-to-find-corrupt-files-periodically)
-  * [What is the meaning of the leading 11 characters (e.g. "cf...p.....") in my snapshot logs?](#what-is-the-meaning-of-the-leading-11-characters-eg-cfp-in-my-snapshot-logs)
+   * [Does _Back in Time_ support full system backups?](#does-_back-in-time_-support-full-system-backups)
```
Comment (buhtz) [inline FAQ.md]: Remove the `_` in the URL.
`-_back-in-time_-` -> `-back-in-time-`

File: FAQ.md [hunk 2]
```diff
@@ -1,51 +1,100 @@
...
+   * [Does _Back in Time_ support backups one cloud storage like OneDrive or Google Drive?](#does-_back-in-time_-support-backups-one-cloud-storage-like-onedrive-or-google-drive)
```
Comment (buhtz) [inline FAQ.md]: Remove the `_` in the URL.
`-_back-in-time_-` -> `-back-in-time-`

File: FAQ.md [hunk 3]
```diff
@@ -1,51 +1,100 @@
...
+   * [How to use QNAP QTS NAS with BIT over SSH](#how-to-use-qnap-qts-nas-with-bit-over-ssh)
+   * [How to use Synology DSM 5 with BIT over SSH](#how-to-use-synology-dsm-5-with-bit-over-ssh)
+   * [How to use Synology DSM 6 with BIT over SSH](#how-to-use-synology-dsm-6-with-bit-over-ssh)
+   * [How to use Western Digital MyBook World Edition with BIT over ssh?](#how-to-use-western-digital-mybook-world-edition-with-bit-over-ssh)
+- [Uncategorized questions](#uncategorized-questions)
+   * [Version >= 1.2.0 works very slow / Unchanged files are backed up](#version--120-works-very-slow--unchanged-files-are-backed-up)
+   * [Which additional features on top of a GUI does BIT provide over a self-configured rsync backup? I saw that it saves the names for uids and gids, so I assume it can restore correctly even if the ids change. Great! :-) Are there additional benefits?](#which-additional-features-on-top-of-a-gui-does-bit-provide-over-a-self-configured-rsync-backup-i-saw-that-it-saves-the-names-for-uids-and-gids-so-i-assume-it-can-restore-correctly-even-if-the-ids-change-great-are-there-additional-benefits)
```
Comment (buhtz) [inline FAQ.md]: Modify the URL. The emoticon is not correct interpreted by the toc generator

`if-the-ids-change-great-are-there-additional-benefits` -> `if-the-ids-change-great---are-there-additional-benefits`

Comment (buhtz) [reply FAQ.md]: I am not able to point the comment to the correct line. Am I wrong? ;)
It is line 39

**Comment by aryoda** [comment]
@buhtz THX a lot for finding the broken TOC anchor links in the FAQ (sharp eyes :-)

I have fixed them now and tested in my cloned repo so it should be OK now...

**Comment by buhtz** [comment]
It wasn't my "sharp eyes", just my robotic fingers clicking on each of that links because I don't trust that TOC generators. I have same problems on Codeberg.org (Gitea/Forgjo based) with several other TOC generators.

**Review by buhtz — APPROVED** [review]

**Pull request merged**